integration_test.go

```diff
@@ -163,6 +163,16 @@ func TestIntegrationRoundTrip(t *testing.T) {
 							return
 						}
 
+						if msg.DeliveryTag == nil {
+							receiveErr = fmt.Errorf("Error after %d receives: nil deliverytag received", i, err)
+							return
+						}
+
+						if msg.DeliveryTag != nil && len(msg.DeliveryTag) != 16 {
+							receiveErr = fmt.Errorf("Error after %d receives: deliverytag should be 16 length byte array representing a UUID. Got: %v", i, err, len(msg.DeliveryTag))
+							return
+						}
+
```

Comment:
The number of arguments doesn't match the formatters in the two calls to `fmt.Errorf` here. Looks like it's just the extraneous `err` arguments.

Reply:
Sorry could of silly mistakes here, will fix up.